interface/modules/custom_modules/oe-module-weno/sql/table.sql

```diff
@@ -38,52 +34,48 @@ CREATE TABLE `weno_pharmacy` (
   `Modified` datetime DEFAULT NULL,
   `Deleted` datetime DEFAULT NULL,
   `24HR` varchar(3) DEFAULT NULL,
-  `on_weno` tinytext NOT NULL,
+  `on_weno` tinytext,
   PRIMARY KEY (`id`),
-  UNIQUE KEY `id` (`id`),
   UNIQUE KEY `ncpdp` (`NCPDP`)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+) ENGINE=InnoDB;
 #EndIf
 
 #IfNotTable weno_assigned_pharmacy
-CREATE TABLE IF NOT EXISTS `weno_assigned_pharmacy` (
+CREATE TABLE `weno_assigned_pharmacy` (
     `id` INT(10) NOT NULL AUTO_INCREMENT,
     `pid` BIGINT(20) NOT NULL,
     `primary_ncpdp` VARCHAR(8) NOT NULL,
     `alternate_ncpdp` VARCHAR(8) NOT NULL,
-    PRIMARY KEY(id)
+    FOREIGN KEY (`pid`) REFERENCES `patient_data`(`pid`),
```

Comment:
Can't do this. One reason is patient_data doesn't use a primary key. In the past mysql would be silent in reporting this as an error. So in this case your key will be generated however, the reference you hoped for will simply be ignored.
Best to fix so future folks looking at this won't go away with a misconception.